babel.config.js

```diff
@@ -1,6 +1,11 @@
 module.exports = {
   presets: [
-    '@babel/preset-react',
+    [
+      '@babel/preset-react',
+      {
+        runtime: 'automatic'
```

Comment:
Why?

Reply:
This is required for the last react version :)